packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -2496,7 +3618,7 @@ test.serial('writeGranuleFromApi() removes preexisting granule file from postgre
   });
   const returnedGranule = await granulePgModel.create(knex, pgGranule, '*');
 
-  const [fakeFile] = await filePgModel.create(knex, {
+  const fakeFile = await filePgModel.create(knex, {
```

Comment:
This doesn't seem correct - `.create` returns an array of type objects.

Reply:
hmm, I can change this back. I did not make this change, but can try and investigate when/how it got in if helpful.